storage/innobase/row/row0ins.cc

```diff
@@ -1568,6 +1573,74 @@ class ib_dec_in_dtor {
 	ulint&		counter;
 };
 
+/*********************************************************************//**
+Reads sys_trx_end field from clustered index row.
+@return trx_id_t */
+static
+trx_id_t
+row_ins_get_sys_trx_end(
+/*===================================*/
+			const rec_t *rec,	/*!< in: clustered row */
+			ulint *offsets,		/*!< in: offsets */
+			dict_index_t *index)	/*!< in: clustered index */
+{
```

Comment:
Do not add `/******/` or `/*====*/ comments. Indent parameters by one line. Why is `offsets` not a pointer to `const`?